R/helpers_kintr_Increments.R

```diff
@@ -0,0 +1,318 @@
+#' @importFrom knitr knit_print
```

Comment:
I think we should only have this statement once, so suggest to look for the best place and then have it there once

Reply:
Ultimately, when we merge all the class-specific `knit_print_<ClassName>.R` files into one, we will have only one instance.  I'll check to see if we can get away with only one in the mean time.

Reply:
hm I would not merge into a huge file in the end, it will be too much at once. could also move one statement to the central package.R file

Reply:
Good idea.  Have placed it in `crmPack-package.R`.